src/utils/config.py

```diff
@@ -0,0 +1,47 @@
+import sys
+from dataclasses import dataclass, asdict
+import json
+from typing import Dict, List, Tuple
+from src.utils import pot
+
+
+@dataclass
+class Config:
+    # gui atributi
+    barve: Dict[str, List[int]]
+    gui_velikost: Dict[str, int]
+    ime_igralca: str
+    naslov_igre: str
+    font_type: str
+    font_size: int
+    meni_ime: str
+    default_ime: str
+    meni_igraj: str
+    meni_config: str
+    meni_izhod: str
+    theme_color: str
+    # vesolje atributi
+    pozicija_ladje_x: int
+    pozicija_ladje_y: int
+    velikost_ladje_x: int
+    velikost_ladje_y: int
+    hitrost_ladje: int
```

Comment:
Vse stvari ki se tičejo configuracije ladje združi v svoj dataclass... In enako za menu, font etc...

Reply:
@urosjarc Kaj to pomeni potem, da moram ustvarit tudi za vsak class posebej config.json, class Config_meni ->config_meni.json, class Config_fonti -> config_fonti.json ? Ali nekak trenutno kodo config.py prilagodim, da se bo vseeno vse črpalo samo iz config.json ?

Reply:
Najbolje je da obstaja samo en vir configuracije.
> https://stackoverflow.com/questions/51564841/creating-nested-dataclass-objects-in-python

Reply:
🤷‍♂️

Reply:
Ko boš ustvarjal nestano configuracijo je potrebno bit malce spreten ko boš iniciliziral notranjo strukturo classa saj je potrebno narediti inicilizacijo notranjih objektov v post initu parent classa. Če ti ne bo ratalo danes tega narediti se bova dobila za 10 min da ti pokažem.

Reply:
:smile: Ti bom pomagal ko končam z inštrukcijami prav :+1:

Reply:
Moram vse to sam fajn predelat prej...te pokličem če/ko bom obupau... 👍